### PR TITLE
sslscan: init at 1.11.0

### PR DESCRIPTION
**Pull request by fpletz**

### CODE REVIEW

File: pkgs/tools/security/sslscan/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  installPhase = ''
+    make install BINPATH=$out/bin MANPATH=$out/share/man
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Tests SSL/TLS services and discover supported cipher suites";
+    homepage = https://github.com/rbsec/sslscan;
+    license = licenses.gpl3;
+    maintainers = with maintainers; fpletz;
```
Comment (pSub) [inline pkgs/tools/security/sslscan/default.nix]: `maintainers` is a list.

File: pkgs/tools/security/sslscan/default.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    platforms = with platforms; all;
```
Comment (pSub) [inline pkgs/tools/security/sslscan/default.nix]: Why not `platforms = platforms.all`? Should be more efficient.

File: pkgs/tools/security/sslscan/default.nix [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+  src = fetchurl {
+    url = "https://github.com/rbsec/sslscan/archive/${version}-rbsec.tar.gz";
+    sha256 = "19d6vpcihfqs35hni4vigcpqabbnd3sndr5wyvfsladgp40vz3b9";
+  };
+
+  buildInputs = [ openssl ];
+
+  installPhase = ''
+    make install BINPATH=$out/bin MANPATH=$out/share/man
```
Comment (pSub) [inline pkgs/tools/security/sslscan/default.nix]: There is `installFlags` which can be used to set BINPATH and MANPATH.

**Comment by fpletz** [comment]
@pSub Thanks for the review. All issues fixed & tested.

**Comment by pSub** [comment]
That was quick! :-)

**Pull request merged**